cristin-project/src/test/java/no/unit/nva/cristin/projects/projectQuery.feature

```diff
@@ -160,16 +162,34 @@ Feature: API tests for Cristin projects query
 
   Scenario: Query accepts organization uri as query parameter
     Given path '/project/'
-    * def organizationId = 'https://api.dev.nva.aws.unit.no/cristin/organization/20202.0.0.0'
     And param query = queryString
     And param organization = organizationId
     When method GET
     Then status 200
 
   Scenario: Query with illegal organizationId  returns 400 Bad request
     Given path '/project/'
-    * def organizationId = 'htttps:/api.dev.nva.aws.unit.no/cristin/organization/20202.0.0.0'
+    * def illegalOrganizationId = 'htttps:/api.dev.nva.aws.unit.no/cristin/organization/20202.0.0.0'
     And param query = queryString
-    And param organization = organizationId
+    And param organization = illegalOrganizationId
     When method GET
     Then status 400
+
+  Scenario Outline: Query accepts status parameter value case independent
```

Comment:
What happens if the query contains > 1 instances of the parameter?

Reply:
Last parameter is used..  RequestInfo uses HashMap to keep parameters